pkg/server/server.go

```diff
@@ -76,11 +78,38 @@ func NewServer(service storage.Service, options ...server.Option) (*http.ServeMu
 	return mux, nil
 }
 
+type ServerInfo struct {
+	ID    string    `json:"id"`
```

Comment:
nit: worth calling this `DID` instead?

Reply:
I would prefer ID - this is the server identity (it's "id"), the type of the ID happens to be DID...